masu/external/accounts_accessor.py

```diff
@@ -58,7 +64,22 @@ def _set_source(self):
 
         return None
 
-    def get_accounts(self):
+    @staticmethod
+    def is_polling_account(account):
```

Comment:
I'm not sure this makes sense as a staticmethod when the rest of the class expects to be instantiated. This function references none of the class resources, so putting it into this class adds no real value that I can see.

To me, this would be better off as either 1) as a simple utility function in this module outside of the class or 2) In a static class with no instance methods.

(I know we have other staticmethods that are similarly placed into classes. If there's no objections, I'd like to reconcile those at some point, too.)

Reply:
I'll update the comment to be more descriptive.  But I put it here because it takes an argument that the `AccountsAccessor().get_accounts()` method returns.